README.md

```diff
@@ -1,119 +1,229 @@
-| [firebase_auth][auth_pub] | ![pub package][auth_badge] | [Firebase Authentication][auth_product] | [`firebase_auth`][auth_code] | <img src="https://user-images.githubusercontent.com/394889/70172910-9174c880-1686-11ea-8e2e-3d8c1cc78cce.png" alt="hummingbird" width="25"> | macOS |
-| [firebase_core][core_pub] | ![pub package][core_badge] | [Firebase Core][core_product] | [`firebase_core`][core_code] | <img src="https://user-images.githubusercontent.com/394889/70172910-9174c880-1686-11ea-8e2e-3d8c1cc78cce.png" alt="hummingbird" width="25"> | macOS |
-| [firebase_crashlytics][crash_pub] | ![pub package][crash_badge] | [Firebase Crashlytics][crash_product] | [`firebase_crashlytics`][crash_code] | | macOS |
-| [firebase_database][database_pub] | ![pub package][database_badge] | [Firebase Realtime Database][database_product] | [`firebase_database`][database_code] | | |
-| [firebase_dynamic_links][dynamic_links_pub] | ![pub package][dynamic_links_badge] | [Firebase Dynamic Links][dynamic_links_product] | [`firebase_dynamic_links`][dynamic_links_code] | | |
-| [firebase_in_app_messaging][in_app_messaging_pub] | ![pub package][in_app_messaging_badge] | [Firebase In-App Messaging][in_app_messaging_product] | [`firebase_in_app_messaging`][in_app_messaging_code] | | |
-| [firebase_messaging][messaging_pub] | ![pub package][messaging_badge] | [Firebase Cloud Messaging][messaging_product] | [`firebase_messaging`][messaging_code] | | |
-| [firebase_ml_vision][ml_vision_pub] | ![pub package][ml_vision_badge] | [Firebase ML Kit][ml_vision_product] | [`firebase_ml_vision`][ml_vision_code] | | |
-| [firebase_performance][performance_pub] | ![pub package][performance_badge] | [Firebase Performance Monitoring][performance_product] | [`firebase_performance`][performance_code] | | |
-| [firebase_remote_config][remote_config_pub] | ![pub package][remote_config_badge] | [Firebase Remote Config][remote_config_product] | [`firebase_remote_config`][remote_config_code] | | |
-| [firebase_storage][storage_pub] | ![pub package][storage_badge] | [Firebase Cloud Storage][storage_product] | [`firebase_storage`][storage_code] | | macOS |
+## Plugins
+
+**Table of contents:**
+
+ - [AdMob (`firebase_admob`)](#firebase_admob)
+ - [Analytics (`firebase_analytics`)](#firebase_analytics)
+ - [Authentication (`firebase_auth`)](#firebase_auth)
+ - [Cloud Firestore (`cloud_firestore`)](#cloud_firestore)
+ - [Cloud Functions (`cloud_functions`)](#cloud_functions)
+
+ - [Cloud Messaging (`firebase_messaging`)](#firebase_messaging)
+ - [Cloud Storage (`firebase_storage`)](#firebase_storage)
+
+ - [Core (`firebase_core`)](#firebase_core)
+
+---
+
+### `firebase_admob`
+
+> [![firebase_admob][admob_badge_pub][admob_pub]
```

Comment:
```suggestion
> [![firebase_admob][admob_badge_pub]][admob_pub]
```

Reply:
Aye not finished yet :) still need to do all the other plugins

Reply:
Haha no problem - just a coincidence that I looked at it 😄

Reply:
I should have left it as a daft so my bad :D thanks for reviewing though, appreciated